File: pkg/codegen/testing/test/testdata/throw-not-implemented-pp/dotnet/throw-not-implemented.cs
using System.Collections.Generic;
using Pulumi;

	
static object NotImplemented(string errorMessage) 
{
    throw new System.NotImplementedException(errorMessage);
}

return await Deployment.RunAsync(() => 
{
    return new Dictionary<string, object?>
    {
        ["result"] = NotImplemented("expression here is not implemented yet"),
    };
});

File: pkg/codegen/testing/utils/testdata/throw-not-implemented-pp/go/throw-not-implemented.go
package main

import (
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func notImplemented(message string) pulumi.AnyOutput {
	panic(message)
}

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		ctx.Export("result", pulumi.Any(notImplemented("expression here is not implemented yet")))
		return nil
	})
}
